docsite/source/settings.html.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Settings
+layout: gem-single
+name: dry-system
+---
+
+## Basic usage
+
+Dry-system provides a built-in `:settings` component which you can use in your application. In order to set it up, simply define a bootable `:settings` component and import it from the `:system` provider:
```

Comment:
```suggestion
dry-system provides a built-in `:settings` component which you can use in your application. In order to set it up, simply define a bootable `:settings` component and import it from the `:system` provider:
```

Tim said it's OK to use the actual, lower-cased, gem name even in the beginning of a sentence 😄

Reply:
😄